pyproject.toml

```diff
@@ -14,6 +14,12 @@ authors = [
 python = "^3.8"
 transformers = "^4.4.2"
 tensorflow = "^2.4.1"
+jupyter = "^1.0.0"
+torch = "^1.8.0"
```

Comment:
i think it should be optional or something? so it will be not installed when our package is installed, as installing torch is not as simple as pip install torch in some cases

Reply:
Torch is in requirements by flair.  In case of installation I tottaly agree with that, as Installing torch with GPU support is not that easy.  (Different cuda versions) Maybe some config for CPU/GPU installation? We should pick some solution,